CONTRIBUTING.md

```diff
@@ -115,6 +115,10 @@ To deploy the plugin and test code onto a test environment we support a set of
 Makefile targets. There targets rely on environment variables to point to the
```

Comment:
As you are here to update .. small request to correct the typo: `There ` => `These `